### DISCUSSION TITLE
Is "Enter QR Code" button still needed in header section if "Search or enter qr code" input field provides same functionality ?

### DISCUSSION THREAD

**Discussion by devendra-tarento** [Sunbird-Ed/Community#362] [General Discussion]
**Search or enter qr code** input field in header section provides capability to search the content by entering text or qr code and **Enter QR Code** button in header section also provides the functionality to search the content by entering qr code.

In joyful theme we don't see **Enter QR Code** button anywhere in the header section and it could be due to **Search or enter qr code** input field, which provides the same capability.

But in Classic Theme we still see **Enter QR Code** button along with **Search or enter qr code** input field.

Now my doubt is why do we need  **Enter QR Code** button in classic theme if **Search or enter qr code** input field provides the same functionality ?

Can't we simply remove **Enter QR Code** button in classic theme too ? (As it is already removed in Joyful Theme).

Please correct me if my understanding is wrong.
Attached the video of the above explanation for more clarity.

https://user-images.githubusercontent.com/114464145/213426726-d0433487-250b-4e0c-af63-4ef8e4d9ec7d.mp4

@vinukumar-vs @Krishnaj20 @rajeevsathish

**Comment by Krishnaj20** [comment]
@devendra-tarento - you are correct, we do not need this in the classic theme. This looks like was a functional debt item which was not prioritized. 

@rajeevsathish - are there any other similar items in the classic theme that are obsolete which need to be addressed?

**Reply by vinukumar-vs** [reply]
@Krishnaj20 we have to take a call based on usage. If the feature was not used in classic them, then we can completely remove it. 
If the users are using this, how they will get to know they can user search input itself for QR code.

We can take a call to remove it fully or not.

@devendra-tarento we can disable this easily using styles the way it was handled in joyful theme. 

Thanks for the contribution & we will close it based on our analysis. 
@rajeevsathish @sri-pusuluri

**Reply by devendra-tarento** [reply]
@vinukumar-vs we don't see any Enter QR Code Button Element in the Joyful theme code which is made hidden through styles. Following same practice we deleted it from Classic Theme.  Please confirm still you want it be made hidden from style we can do that.

@rajeevsathish @Krishnaj20

**Reply by Krishnaj20** [reply]
@vinukumar-vs - what are the next steps on this contribution?

**Reply by devendra-tarento** [reply]
@vinukumar-vs Please provide your input on this so that we can make progress accordingly.

@rajeevsathish @Krishnaj20
